configure.ac

```diff
@@ -38,6 +38,15 @@ MONO_VERSION_MAJOR=`echo $VERSION | cut -d . -f 1`
 MONO_VERSION_MINOR=`echo $VERSION | cut -d . -f 2`
 MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 
+# Temporary workaround for Apple Silicon
+if uname -a | grep -q 'Darwin.*arm64'; then 
+    if test $ac_cv_host = arm-apple-darwin20.0.0 -o $ac_cv_target = arm-apple-darwin20.0.0; then 
+       echo "You are running on Apple Silicon, but using an old config.guess, invoke configure like this:"
+       echo "Run configure using ./configure --host=i686-w64-mingw32 or --host=x86_64-w64-mingw32"
+       exit 1
+    fi
+fi
```

Comment:
I'd prefer to move this to after line 78/88 so it is together with the existing cygwin check